configure.ac

```diff
@@ -26,6 +26,12 @@ GENERIC_MAJOR_VERSION=$AX_MAJOR_VERSION
 GENERIC_MINOR_VERSION=$AX_MINOR_VERSION
 GENERIC_MICRO_VERSION=$AX_POINT_VERSION
 
+# Defines needed by Environment.C (formerly in queso.h)
+# The values are not quoted strings per queso.h history (Environment.C uses for arithmetic)
+AC_DEFINE_UNQUOTED([GENERIC_MAJOR_VERSION], ${GENERIC_MAJOR_VERSION}, "Major version")
+AC_DEFINE_UNQUOTED([GENERIC_MINOR_VERSION], ${GENERIC_MINOR_VERSION}, "Minor version")
+AC_DEFINE_UNQUOTED([GENERIC_MICRO_VERSION], ${GENERIC_MICRO_VERSION}, "Micro version")
```

Comment:
Yes, that looks correct.  I'm not sure how I missed that.  I can push that change.